gordon_gcp/schemas/audit-log.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
There's drafts 6 and 7 out, but `jsonschema` only supports 3 and 4 at the moment.